doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+So for each detected peak in a parent footprint, the minimum of each pixel and the pixel on the opposite side of the peak is used to create a symmetric template that will be used to deblend the sources, which is equivalent to::
+
+    symmetric = np.min([image, image[::-1, ::-1]], axis=0)
+
+for a given image ``array``.
+
+For example, let's look at a small blend from the HSC COSMOS dataset:
+
+.. image:: images/colorBlend.png
+
+Or, for now, just the r-band image
+
+.. image:: images/redBlend.png
+
+This is a good blend to explore because the resulting templates show us a variety of success and failure modes.
+Looking first at source 0, which is bright and fairly isolated, the symmetric template is:
+
+.. image:: images/template0.png
+
+Even in this case the template isn't perfect (and isn't intended to be), which we see by looking at the residual of the image with the source 0 template subtracted:
+
+.. image:: images/templateResidual0.png
+
+Most of the residual that is part of the actual source is due to the non-symmetric shape of the PSF, however there is also a small blob to the right due to an undetected source that will eventually be falsy included in the flux of this source when we re-apportion the flux in the image (see `re-apportion`_).
+
+Looking at source 2 we see another artifact caused by a complicated galaxy that has multiple non-monotonic regions that either need to be modeled as additional components or a more complex non-monotonic model (see :ref:`scarlet_restart`).
+
+.. image:: images/template2.png
+
+These, and other types of "unrecognized blends" are unavoidable and can be expected in all ground based surveys, including HSC and LSST.
+
+Finally we have source 3, which shows the worst deblending case for a symmetry based deblender: three sources that are nearly aligned and equally spaced
+
+.. image:: images/template3.png
+
+Here, the template for the fainter source in the center picks up flux from both of its neighbors, which will cause it to steal flux from both of them and affect all three models once we re-apportion the flux.
+Attempts have been made to use a monotonic projection for templates to mitigate this problem but those tend to cause large fractal-like structures that have been determined to make the templates worse in general, so they are not used in production (but using them does exist via a config option).
+
+PSF Templates
+^^^^^^^^^^^^^
+
+Sources with very low signal to noise (like all sources labeled 5 and greater in the above blend) are unlikely to yield very good symmetric templates because the models are dominated by noise and quickly truncate.
+So :py:mod:`~lsst.meas.deblender` attempts to fit all sources to the PSF (including fitting the centroid and a linear background model).
+Any sources with a sufficiently small residual (determined by the task config) will use the PSF template in place of the symmetric template.
```

Comment:
I was trying to avoid making the `meas_extensions_scarlet` docs contain too much information about `meas_deblender`, but I guess having a few config parameters listed explicitly isn't the worst thing.